### ISSUE TITLE
Superfish Keyboard Navigation Improvements

### ISSUE THREAD

**Issue by joewhitsitt** [uiowa/uiowa#3517]
Split from https://github.com/uiowa/uiowa/issues/1120
## Issue

- User becomes lost in sub-menu—keyboard pattern?
- Keyboard pattern? Can these elements be focusable by tab; child elements by directionals?

**Comment by richardbporter** [comment]
It seems like SF does not clone menu parents on tab events which means keyboard users cannot get to the actual link. We attempted to fix this in #3527 but then realized this issue was split out. We started with 1ae499413d27787b5d2bc06c2428ff671daf5f57 but it needs more thought/testing.

**Comment by GaryRidgway** [comment]
With this as well, when tabbing in to the menu we should probably only scope the click event to fire if the menu is not already open.
